src/styles/index.scss

```
@@ -1,3 +1,163 @@
+$card-width: 200px;
+$imac-width: 160px;
+$imac-margin: ($card-width - $imac-width) / 2;
+$desc-width: 166px;
```

No need to hardcode width for the content, use 100% instead
```suggestion
$desc-width: 100%;
```